app/activeplan/homepage/contactArrangements/post.controller.js

```diff
@@ -0,0 +1,43 @@
+const { body } = require('express-validator')
+const { logger } = require('../../../../common/logging/logger')
+const { setSentencePlanComment } = require('../../../../common/data/sentencePlanningApi')
+const { getContactArrangements } = require('./get.controller')
+
+const validationRules = () => {
+  return [
+    body('contactArrangements')
+      .isLength({ min: 1 })
+      .withMessage('Enter contact arrangements'),
+    body('contactArrangements')
+      .trim()
+      .escape(),
```

Comment:
body('contactArrangements', 'Enter contact arrangements')
      .isLength({ min: 1 })
      .trim()
      .escape(),

Reply:
I quite like keeping the validation and sanitisation separate but no biggy. Changed.